README.md

```diff
@@ -13,12 +13,28 @@ The consumer will periodically run to seed a mongo database from the HackerNews
 
 To run the consumer:
 
+The command below relies on a running mongo database running
+
 ```bash
 cp .env.example app.env && go run cmd/consumer/main.go
 ```
 
-Remember to remove the `app.env` file:
+Remember to remove the `app.env` file once you are done:
 
 ```bash
 rm app.env
-```
\ No newline at end of file
+```
+
+To run a fully functional consumer use the following command:
+
+```bash
+docker-compose up -d
+```
+
+**_Note: Ctrl-C command on just `docker-compose up` seemed to hang, but running in the background and
```

Comment:
If the program is hanging on `Ctrl-C` it's likely there's a blocking channel somewhere that's not allowing the program to exit.

Reply:
Interestingly enough the docker containers are all stopped and in exited state. I'd expect the same blocking in background mode when running docker-compose down as hitting Ctrl-c in tail mode (docker-compose up)

Reply:
I figured it out, the restart key in mongo was causing the hang. I've removed it as now works with just docker-compose up